cropduster/views/__init__.py

```diff
@@ -229,7 +234,12 @@ def upload(request):
     if six.PY2 and isinstance(orig_file_path, unicode):
         orig_file_path = orig_file_path.encode('utf-8')
     orig_image = get_relative_media_url(orig_file_path)
-    img = PIL.Image.open(orig_file_path)
+    file = default_storage.open(orig_file_path)
```

Comment:
It looks like this works in Pillow (just tested)

```python
with default_storage.open(orig_file_path) as f:
    im = PIL.Image.open(f)
    im.filename = orig_file_path
```

That handles the close. Here's what `help(PIL.Image.open)` has to say about the first parameter (and why I thought it would work to pass it an instance of `django.core.files.base.File`:

> A filename (string), pathlib.Path object or a file object. The file object must implement file.read, file.seek, and file.tell methods, and be opened in binary mode.

The context manager for the file as `self.close()` in its `__exit__`